game.rb

```diff
@@ -52,6 +52,7 @@ def initialize(strategy:)
 
     @strategy = strategy
     super()
+    identify!
```

Comment:
I see that the original code had a special case for the seer when [identifying players](https://github.com/mtcmorris/werewolf-sim/commit/41830521aa2bdc98d5bfe90b9b10a0d6f61ab161#diff-feb2bfdaacec4537f8d5fdcebffeb383R142) and [revealing](https://github.com/mtcmorris/werewolf-sim/commit/41830521aa2bdc98d5bfe90b9b10a0d6f61ab161#diff-feb2bfdaacec4537f8d5fdcebffeb383R142).

However, I feel identifying the seer from the start like this is a bit more intuitive. "Identified" basically means "the seer knows what side they're on", and I think this should avoid bugs with special-casing for the seer.